.github/workflows/e2e-preset-tuning-test.yml

```diff
@@ -0,0 +1,136 @@
+name: E2E Preset tuning Test
+
+on:
+    workflow_run:
+        workflows: ["Build and Push Preset Models"]
+        types:
+            - completed
+    workflow_dispatch: {}
+
+env:
+    GO_VERSION: "1.22"
+
+permissions:
+    id-token: write
+    contents: read
+
+jobs:
+  e2e-preset-tuning-tests:
+    needs: determine-models
+    if: github.event_name == 'workflow_dispatch' || github.event.workflow_run.conclusion == 'success'
+    runs-on: ubuntu-latest
+    environment: preset-env
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.2.2
+        with:
+            submodules: true
+            fetch-depth: 0
+
+      - name: 'Az CLI login'
+        uses: azure/login@v2.2.0
+        with:
+            client-id: ${{ secrets.AZURE_CLIENT_ID }}
+            tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+            allow-no-subscriptions: true
+
+      - name: 'Set ACR Subscription'
+        run: az account set --subscription ${{secrets.AZURE_SUBSCRIPTION_ID}}
+
+      - name: Set up kubectl context
+        run: |
+          az aks get-credentials --resource-group llm-test --name GitRunner
+
+      - name: Get test meta
+        id: get_test_meta
+        run: |
+            echo "TAG=0.0.7" >> $GITHUB_OUTPUT
+            echo "NODEPOOL_NAME=tuning" >> $GITHUB_OUTPUT
+            echo "NODE_COUNT=1" >> $GITHUB_OUTPUT
+            echo "NODE_VM_SIZE=Standard_NC6s_v3" >> $GITHUB_OUTPUT
+            echo "NODE_OSDISK_SIZE=100" >> $GITHUB_OUTPUT
```

Comment:
similar to the e2e-preset-test either in this pr or the next, update this to use a config file with constants